build/yamls/antrea.yml

```diff
@@ -38,6 +38,124 @@ spec:
     served: true
     storage: true
 ---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  labels:
+    app: antrea
+  name: clusternetworkpolicies.crd.antrea.tanzu.vmware.com
```

Comment:
i am open to using a different subgroup naming instead of "crd".. but at the least i think we should group all user facing CRDs in a different subgroup..

Reply:
I feel adding "crd." sounds strange. We might like to make the user facing CRDs/API more convenient to consume. If we want to separate CRDs and Controller API for Agent, maybe we should add a sub-group for the latter?

Reply:
id like to close on this topic soon.. so to simplify for users, all user facing crds will be directly under antrea.tanzu.vmware.com? based on that ill make the change to create crd with group "clusternetworkpolicies.antrea.tanzu.vmware.com"

Reply:
How about let us use clusternetworkpolicies.antrea.tanzu.vmware.com for now. And let us continue the API group discussion. You can make a proposal for all resources if you have some ideas.

Reply:
thanks Jianjun. that sounds good to me. Will update and we can discuss a proposal in the upstream meeting